Kitodo-DataManagement/src/main/resources/db/migration/V2_62__Rename_the_table_folder_to_subfolder_type.sql

```diff
@@ -0,0 +1,20 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+--
+-- Migration: Rename the table folder to subfolder type
+--
+-- 62. Rename the table folder to subfolder type
+--
+
+USE kitodo;
+CREATE TABLE subfolder_type AS SELECT * FROM folder;
```

Comment:
Why are you not using an `ALTER TABLE` statement for renaming an existing table like `ALTER TABLE folder RENAME TO subfolder_type`?

Reply:
Forget to mention that existing foreign key(s) to this and from this table must be updated.